web/app/scripts/app.js

```diff
@@ -31,6 +31,30 @@
     }
 
     /* ngInject */
+    function TranslationConfig($translateProvider, $windowProvider) {
+        // Configure the location of the translation json files on the server
+        $translateProvider.useStaticFilesLoader({
+            prefix: 'i18n/',
+            suffix: '.json'
+        });
+
+        // Log untranslated tokens to console
+        $translateProvider.useMissingTranslationHandlerLog();
+
+        // Enable sanitization of token
+        $translateProvider.useSanitizeValueStrategy('sanitizeParameters');
+
+        // Get the browser's navigator object and use for determining preferred language
+        var navigator = $windowProvider.$get().navigator;
+        var language = (navigator.languages ? navigator.languages[0] :
+                        (navigator.language || navigator.userLanguage)).substring(0,2);
+
+        // Interpret Zulu ('zu') to request dev language ('exclaim')
+        $translateProvider.preferredLanguage(language === 'zu' ? 'exclaim' : language);
```

Comment:
I've never been entirely comfortable with the selection of Zulu as a debug language given that it has ~10 million native speakers. Looking at the list of valid language codes, I think that Volapük (`vo`), Pāli (`pi`), Old Church Slavonic (`cu`), Interlingue (`ie`), or Avestan (`ae`) would be better choices. Most of them are primarily used as liturgical languages, which makes it relatively unlikely that someone would actually want to use the site in that language. Interlingue is a constructed language, similar to Esperanto but significantly less popular.